src/screens/DateFilterScreen/Header.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from "react";
+import { View } from "react-native";
+import CommonHeader from "../../components/Header";
+import ActionButton from "../../components/ActionButton";
+
+type Props = {
+  onCancel: () => void,
+  onReset: () => void,
+  dateRange: string,
+  title: string,
+  titleLabel: string
+};
+
+const Header = ({ onCancel, onReset, dateRange, title, titleLabel }: Props) => (
+  <View>
+    <CommonHeader
+      leftElement={<CommonHeader.BackButton onPress={onCancel} />}
+      title={title}
+      titleLabel={titleLabel}
+      titleAccesible
+      titleAccessibilityTraits={["header"]}
```

Comment:
The `CommonHeader` already has the `header` trait. What are the benefits of repeating it here?